profiler/include/profile_grouped_gemm_impl.hpp

```diff
@@ -7,51 +7,29 @@
 
 #include "ck/ck.hpp"
 #include "ck/tensor_operation/gpu/device/tensor_layout.hpp"
-#include "ck/tensor_operation/gpu/device/device_gemm.hpp"
+#include "ck/tensor_operation/gpu/device/device_grouped_gemm_xdl.hpp"
```

Comment:
minor issue: only need "device_grouped_gemm.hpp" not "device_grouped_gemm_xdl_xdl.hpp", since the later is implementation detail and not part of external interface,

Reply:
@zjing14 Please fix this in this PR, since the CI is still WIP